src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -231,7 +231,7 @@ public ObservableList<PieChart.Data> getIncomePieChartData() {
             new PieChart.Data("Gifts", incomePieChartArr[4]), new PieChart.Data("Others", incomePieChartArr[5]));
 
 
-        return incomePieChartData;
+        return FXCollections.unmodifiableObservableList(incomePieChartData);
```

Comment:
made the ObservableList here unmodifiable similar to the expenditure and income list